src/configuration/config-reader.ts

```diff
@@ -1,8 +1,29 @@
+import { BotConfig, generateConfiguration } from "@ubiquibot/configuration";
 import * as fs from "fs";
 import YAML from "yaml";
 import program from "../parser/command-line";
+import { IncentivesConfiguration, validateIncentivesConfiguration } from "./incentives";
 
-const file = fs.readFileSync(program.opts().config, "utf8");
-const configuration = YAML.parse(file);
+let configuration: BotConfig & IncentivesConfiguration;
```

Comment:
I don't think we need `BotConfig` (kernel config) in the plugin, all information needed is provided via workflow dispatch

Reply:
Right, this was written before the latest changes in the configuration where I removed all these value, will remove it.